File: JavaScript/프로그래머스/1/12933. 정수 내림차순으로 배치하기/정수 내림차순으로 배치하기.js
function solution(n) {
    var answer = 0;
    const array = String(n).split("");
    array.sort((a, b) => {
       if (a < b) return 1;
        else return -1;
    }) // 오름 차순
    answer = Number(array.join(""));
    return answer;
}

File: C++/프로그래머스/1/12933. 정수 내림차순으로 배치하기/정수 내림차순으로 배치하기.cpp
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

long long solution(long long n) {
    long long answer = 0;
    string a = to_string(n);
    sort(a.begin(), a.end(), greater<int>());
    answer = stoll(a);
    return answer;
}